VENDORING.md

```diff
@@ -0,0 +1,43 @@
+# Vendoring policies
+
+This document outlines recommended Vendoring policies for Docker repositories.
+(Example, libnetwork is a Docker repo and logrus is not.)
+
+## Vendoring using tags
+
+Commit ID based vendoring provides little/no information about the updates
+vendored. To fix this, vendors will now require that repositories use annotated
+tags to snapshot commits.
+
+Each tag should:
```

Comment:
These requirements are too much for any dependencies. For example, docker/distribution doesn't necessarily define a CHANGLOG.mg, but it has git tags.